packages/frontend/src/app/repository/repository.component.ts

```diff
@@ -90,8 +44,8 @@ export class RepositoryComponent implements OnInit {
 
   @ViewChild(FiltersComponent) filterComponent;
 
-  repoName = '';
-  orgName = '';
+  repoName = ' ';
+  orgName = ' ';
```

Comment:
Curious to know why not the empty string 😁.

Reply:
Oops! I did not mean to change that to a space.  I will revert to the empty string!